exonum/src/runtime/error.rs

```diff
@@ -428,7 +429,7 @@ where
 ///
 /// [`ErrorKind`]: enum.ErrorKind.html
 /// [`CallSite`]: struct.CallSite.html
-#[derive(Clone, Debug, Fail, BinaryValue)]
+#[derive(Clone, Debug, Deserialize, Serialize, Fail, BinaryValue)]
```

Comment:
Implementing `(De)Serialize` is not necessary; see the comment about `ExecutionStatus`.